src/app/donation-start/donation-start.component.ts

```diff
@@ -911,7 +912,11 @@ export class DonationStartComponent implements AfterContentChecked, AfterContent
       this.personIsLoginReady = true;
 
       if (environment.creditDonationsEnabled && person.cash_balance && person.cash_balance[this.campaign.currencyCode.toLowerCase()] > 0) {
-        this.creditPenceToUse = person.cash_balance[this.campaign.currencyCode.toLowerCase()];
+        this.creditPenceToUse = parseInt(
+          person.cash_balance[this.campaign.currencyCode.toLowerCase()].toString() as string,
```

Comment:
@NoelLH why are you casting a number to a string and then back to a number?

Reply:
@alihejazi `parseInt(...)` gives a type error given anything but a string.